src/Generators/DataTablesMakeCommand.php

```diff
@@ -15,17 +15,21 @@ class DataTablesMakeCommand extends GeneratorCommand
      */
     protected $signature = 'datatables:make
                             {name : The name of the datatable.}
-                            {--model : The name of the model to be used.}
+                            {--model= : The name of the model to be used.}
                             {--model-namespace= : The namespace of the model to be used.}
                             {--action= : The path of the action view.}
+                            {--table= : Scaffold columns from the table.}
+                            {--builder : Extract html() to a Builder class.}
+                            {--dom= : The dom of the datatable.}
+                            {--buttons= : The buttons of the datatable.}
                             {--columns= : The columns of the datatable.}';
 
     /**
      * The console command description.
      *
      * @var string
      */
-    protected $description = 'Create a new DataTable service class.';
+    protected $description = 'Create a new dataTable service class.';
```

Comment:
Was that change intentional?

Reply:
Yes, because I thought it's the proper sentence form. Should I revert? Not that good in english. ^_^

Reply:
Well, camelCase looks weird to me. I would either write it all lower case or like the actual package is called: DataTable(s)

Reply:
Good point on camelCase, reverting. :)